### DISCUSSION TITLE
Public IP blocked by public server

### DISCUSSION THREAD

**Discussion by Alejandro-Santillan** [rustdesk/rustdesk#9315] [General]
Good morning

We are having problems in the company, we implemented its Rustdesk Client solution version 1.2.1. It worked for a while but now we are not receiving incoming traffic from their public server 209.250.254.15. We tried connecting a device via cell phone Wi-Fi and the rustdesk client worked normally.
The support@rustdesk.com email does not work.
Does anyone have contact with the team? Could they solve it?

I will appreciate your help.

Regards

**Comment by gsv2003** [comment]
I have some trouble with ip 213.170.76.226
What can I do?! Thx

**Reply by Alejandro-Santillan** [reply]
Hi Stepan
At the moment I have no news from Rustdesk. Everything points to you having to develop your own server. Now we're testing that in the company.
Regards

**Reply by Lilianne-Blaze** [reply]
How many pcs approximately did you have active at your ip?

**Comment by xlionjuan** [comment]
You have large numbers of devices in your company, you should consider build your own server, it won't spent a lot money. Even 5USD VPS is great for most use cases.

**Comment by RR-T** [comment]
I have the same issue... One location I manage just stopped being able to connect one day - I first blamed firewall but after inspecting traffic I can see `rs-ny.rustdesk.com` is resolved to `209.250.254.15`, outgoing packet but no response back to the WAN interface... I've checked some other sites that resolve to the same IP and do receive responses, so it looks like that one static IP was blocked for some reason or another...

**Reply by xlionjuan** [reply]
Build your own server

**Reply by RR-T** [reply]
Whilst that is an option for managing multiple devices in an org, it is not exactly helpful for sporadic connections say - assisting with a relative or friends personal device. Not super user friendly to ask them to install a VPN and configure their client to talk to a self-hosted server.

**Reply by xlionjuan** [reply]
512MB RAM Ubuntu VPS may suit your need.

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/1cr8kfv/should_you_selfhost_a_rustdesk_server/

**Reply by mickeul** [reply]
Hi, this is not what I call a good answer, or not for all situations. When you have to do sporadic assisting for a client or heterogenous park, you can't setup self hosting server for all clients you meet, it is chronophageous et counter-productive.
Can you chekc if you blocked some IP from French ISP Free SAS? If you agree I can communicate you the public IP I suspect being blocked by your public server. (By mail, for security concern, I won't drop it here, so if you can send me your email adress here I'll write you back)
